website_snippet_carousel_product/static/src/js/snippet.options.js

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+odoo.define("website_snippet_carousel_product.snippet_options", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var options = require("web_editor.snippets.options");
+    var wUtils = require("website.utils");
+
+    var _t = core._t;
+
+    options.registry.js_carousel_product = options.Class.extend({
+        popup_template_id: "editor_new_carousel_product_domain",
+        popup_title: _t("Add a Carousel Products"),
+
+        /**
+         * @override
+         */
+        _onLinkClick: function (ev) {
+            var self = this;
+            var $elm = $(ev.target);
```

Comment:
Use `this.$target` instead:

```suggestion
            var $elm = this.$target;
```

Reply:
Is not the same:
- this.$target refers to the snippet
- $elm refers to the menu option

Reply:
Then it's `this.$el`, [check the docs](https://github.com/odoo/odoo/blob/b88900187b87b2ca8c18d712292a367671a1fff5/addons/web_editor/static/src/js/editor/snippets.options.js#L34-L37) (a little bit hidden as always :laughing:).